### ISSUE TITLE
Parametric EQ settings from AutoEQ gives different results

### ISSUE THREAD

**Issue by svenkerst** [ThePBone/DDCToolbox#4]
When importing a ParametricEQ.txt, the plotted graph is quite different from that of AutoEQ itself.

![image](https://user-images.githubusercontent.com/66112297/83236333-a775c880-a193-11ea-8f49-c57b6a7610eb.png)

![image](https://user-images.githubusercontent.com/66112297/83236573-03d8e800-a194-11ea-9cb9-c26b8bf582b3.png)

**Comment by ThePBone** [comment]
Seems right to me, the thin green line and the plot in DDCToolbox look almost identical.

Could you point out which frequency range looks out of place?

**Comment by svenkerst** [comment]
I have the impression that the line is sort of tilted downwards from around 500 Hz onwards. For example, the peaks at 1826 and 7073 have positive amplitudes in the AutoEQ graph, but not when imported into the DDCToolbox. Maybe it makes sense, but I just don't understand why that is.

**Comment by ThePBone** [comment]
Yes, you're right. There is some discrepancy between both graphs. I'll have a  closer look at how AutoEQ calculates their graphs

**Comment by svenkerst** [comment]
Great, thank you! One more thing: shouldn't changing the filter type also change the graph accordingly?

**Comment by ThePBone** [comment]
It does, but you need to click in another table cell to commit your change (the GUI library I'm using handles widgets inside table cells a bit strangely)

Example:
![screencap](https://user-images.githubusercontent.com/38386967/83302182-26eab280-a1fb-11ea-8e41-b70ae8ec4785.gif)

**Comment by ThePBone** [comment]
You're using the latest stable build right? I just checked and your issue seems to be fixed on the latest version directly compiled from sources.
I was rewriting large chunks of the internal filter handling code a while ago so that might have fixed it.

Use the latest nightly release instead:
<https://nightly.timschneeberger.me/ddctoolbox-win/ddctoolbox-win64_1.3.85.zip>

![Screenshot](https://user-images.githubusercontent.com/38386967/83304338-fc025d80-a1fe-11ea-9166-1d31c088ad4a.PNG)

By the way, this version can download profiles directly from AutoEQ (File > Download from AutoEQ)

**Comment by svenkerst** [comment]
Thank you for your help. I was indeed using the latest stable build. The latest nightly build gives more similar results, but still they're a bit different (negative instead of positive 7073 Hz peak for example). Do you know what's the reason behind this?

Another problem I now have with the latest nightly build is that it crashes when I import a VDC file.

**Comment by svenkerst** [comment]
Converting the VDC file to a project file fixed it for me.

**Comment by ThePBone** [comment]
Could you upload the VDC file that crashes the app? Also, you can only import 'classic'-VDCs (VDCs that only contain Peaking filters)
I'll also look into the other issue

**Comment by svenkerst** [comment]
Thank you very much! Here's the VDC file that's causing the toolbox to crash
[Example VDC.zip](https://github.com/ThePBone/DDCToolbox/files/4706274/Example.VDC.zip)

**Comment by ThePBone** [comment]
I couldn't reproduce the crash you described. However, I noticed some VDCs are falsely recognized as not compatible. That issue should be fixed with the latest nightly build: <https://nightly.timschneeberger.me/ddctoolbox-win/ddctoolbox-win64_1.3.90.zip>

**Comment by rlw6534** [comment]
There still seems to be a problem with autoeq downloads and imports.  As you can see the gain levels are significantly different.  The VDC files also sound very different than the convolver files (especially the SE846), so something is wrong:

![image](https://user-images.githubusercontent.com/100954748/156798519-07256472-cfef-44d4-a4fc-ae10e349c5f5.png)


<img width="395" alt="Screen Shot 2022-03-04 at 11 03 52 AM" src="https://user-images.githubusercontent.com/100954748/156797844-915ccf42-1ac3-462b-9820-d4bd317001b1.png">

Note all gains are negative.

and

![image](https://user-images.githubusercontent.com/100954748/156798693-af38e712-bc74-471e-ad8d-2059963459b9.png)


<img width="395" alt="Screen Shot 2022-03-04 at 11 07 44 AM" src="https://user-images.githubusercontent.com/100954748/156798259-3b61ae6f-d1af-40c1-aeb1-9b321a3a5002.png">

Note the treble gains are much higher.

**Comment by nift4** [comment]
Manually adding Low/High Shelf helped for me:
Before (top) vs after (bottom)
![Screenshot_20221010_204142](https://user-images.githubusercontent.com/36458596/194932494-57ddfb47-6071-431d-8ac2-70f578e0fe5b.png)

After vs AutoEQ graph
![Screenshot_20221010_204211](https://user-images.githubusercontent.com/36458596/194932542-539d63bb-1098-4791-9af5-69cbff6dffca.png)
